inventory/byo/hosts.ose.example

```diff
@@ -99,6 +99,14 @@ openshift_release=v3.4
 # modify image streams to point at that registry by setting the following to true
 #openshift_examples_modify_imagestreams=true
 
+# If oreg_url points to a registry requiring authentication, provide the following:
+oreg_host=example.com:443
```

Comment:
Personally I think oreg_host is a little redundant here, we could get the value form oreg_url, maybe we could reuse the logic - https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_examples/defaults/main.yml#L28, no need to ask user to input the oreg_host separately again.